src/frontend/Dockerfile

```diff
@@ -13,7 +13,6 @@ ENV PATH ./node_modules/.bin:$PATH
 
 # install dependencies
 RUN yarn install
-RUN yarn global add react-scripts@3.4.1
```

Comment:
I'm not sure why this is here.  I've removed but didn't properly test.

Reply:
I'm not sure as well, and I'd like to use the nginx template...